File: JavaScript/1040.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => +a);
let [a,b,c,d] = pegarValores(lines.shift())
let media = ((a*2)+(b*3)+(c*4)+(d*1))/10;
let f
if(media >= 7){
    console.log(`Media: ${media.toFixed(1)}`)
    console.log(`Aluno aprovado.`)
}else if(media < 5){
    console.log(`Media: ${media.toFixed(1)}`)
    console.log(`Aluno reprovado.`)
}else{
    console.log(`Media: ${media.toFixed(1)}`)
    console.log(`Aluno em exame.`)
    const pegarValores1 = (line) => line.split(" ").map(a => +a);
    let [e] = pegarValores1(lines.shift())
    console.log(`Nota do exame: ${e.toFixed(1)}`)
    if(e >= 5){
        console.log(`Aluno aprovado.`)
        f = (e+media)/2
        console.log(`Media final: ${f.toFixed(1)}`)
    }else{
        console.log(`Aluno reprovado.`)
        f = (e+media)/2
        console.log(`Media final: ${f.toFixed(1)}`)
    }
}

File: Python/1040.py
lista = (input().split(" "))
a,b,c,d = lista
a = float(a)
b = float(b)
c = float(c)
d = float(d)
media = ((a*2)+(b*3)+(c*4)+d)/10
if(media >= 7):
    print('Media: %.1f'%media)
    print('Aluno aprovado.')
elif(media < 5):
    print('Media: %.1f'%media)
    print('Aluno reprovado.')
else:
    print('Media: %.1f'%media)
    print('Aluno em exame.')
    exame = float(input())
    print('Nota do exame: %.1f'%exame)
    if(exame >= 5):
        print('Aluno aprovado.')
        final = (exame+media)/2
        print('Media final: %.1f'%final)
    else:
        print('Aluno reprovado.')
        final = (exame+media)/2
        print('Media final: %.1f'%final)